src/plugins/discover/public/application/main/services/discover_saved_search_container.test.ts

```diff
@@ -152,6 +119,7 @@ describe('DiscoverSavedSearchContainer', () => {
         ...savedSearch,
         title,
```

Comment:
to make it all green on this test needs to start with a mock
adding
```
      discoverServiceMock.savedSearch.save = jest.fn().mockResolvedValue('123');
```
and if 
```
yarn test:jest --config src/plugins/discover/jest.config.js 
```
can be trusted, Jest tests are satisfied